integration-tests/cypress/commands/api/device.d.ts

```diff
@@ -34,12 +51,23 @@ declare namespace Cypress {
       makeCameraNameTestName?: boolean
     );
 
+
     /**
     * Retrieve device details using name and groupname
+    * use groupId if provided, otherwise groupName
     * compare with expected device details (JSON equivalent to that retunred by API)
     * optioanlly check for a non-200 status code
     */
-    apiCheckDevice(userName: string, cameraName: string, groupName: string, expectedDevice: ComparableDevice, statusCode: number): Chainable<Element>;
+    apiCheckDeviceInGroup(userName: string, cameraName: string, groupName: string, groupId: number, expectedDevice: all, statusCode: number);
+
+
+    /**
+    * Retrieve devices list
+    * compare with expected device details (JSON equivalent to that retunred by API)
+    * pass optional params (params) to API call
+    * optionlly check for a non-200 status code
+    */
+    apiCheckDevices(userName: string, expectedDevice: [ComparableDevice], params: string, statusCode: number);
```

Comment:
ComparableDevice doesn't seem to be defined anymore?

Reply:
now defined in types.d.ts and referenced

Reply:
Sorry, also is `[ComparableDevice]` meant to be an array of devices?  If so, the correct syntax is `ComparableDevice[]`, and if not, the square brackets should go.

Reply:
Yes. Corrected this [] everywhere now